notebooks/corridor/corridor_bucharest.qmd

```diff
@@ -428,7 +431,7 @@ st_write(
 )
 ```
 
-### Define some functions to calculate a different weight
+### Modified approach accounting for distance for initial corridor guess
 
 In this approach, the distance from the initial corridor edges is incorporated into the edge length as a "weight" before calculating the shortest path. This added cost encourages the path to stay as close as possible to the initial corridor. When comparing two edges of the same length, the edge closer to the initial corridor is preferred, as its nodes have smaller distances to the corridor. Conversely, when comparing edges with equal distances to the corridor, the one with the shorter length is selected. Therefore, the "weight" of an edge is determined by its length plus the sum of its distances from the corridor.
```

Comment:
> when comparing edges with equal distances to the corridor, the one with the shorter length is selected.

Why wouldn't the longest edge be selected instead? Wouldn't that be more desirable? In any case, if the distance precision is high enough, such a tie would not occur often, right?

As an alternative, it is mentioned below that one way of calculating the weight is to only use distance, which would overcome the tendency of selecting short lines. This can be done by increasing the proportion of distance from 0.5 to 1. 

Below is a comparison between 0.5 (top) and 1 (bottom) using the full network as described in the last section of the notebook. This change does not show any improvement, but we can check if there is any difference when the real valley is used as input.

![image](https://github.com/user-attachments/assets/dec9ddfe-4da7-4b8d-ad09-2298700937f0)